src/app/features/token/tokenSlice.ts

```diff
@@ -0,0 +1,25 @@
+import { createSlice, PayloadAction } from '@reduxjs/toolkit';
+
+
+interface TokenState {
+    token: string;
+}
+
+// Define the initial state using that type
+const initialState: TokenState = {
+    token: ''
+};
+
+export const tokenSlice = createSlice({
```

Comment:
```suggestion
export const authSlice = createSlice({
```

Refactor all the files to use this name instead.